src/services/api.service.ts

```diff
@@ -416,6 +419,12 @@ export class ApiService implements ApiServiceAbstraction {
         return new SendAccessResponse(r);
     }
 
+
+    async getSendFileDownloadData(send: SendAccessView, request: SendAccessRequest): Promise<SendFileDownloadDataResponse> {
+        const r = await this.send('POST', '/sends/' + send.id + '/access/file/' + send.file.id, request, false, true);
```

Comment:
Isn't `send.id` supposed to be sent base64 encoded or is it already stored this way when concat'd here?

Reply:
No worries, that's why I ask, it's good